tethys_gizmos/static/tethys_gizmos/js/tethys_map_view.js

```diff
@@ -6,6 +6,14 @@
  * LICENSE: BSD 2-Clause
  *****************************************************************************/
 
+ // Backward/forward compatability for OpenLayers
+if (typeof ol.inherits === 'undefined') {
+    ol.inherits = function (child, parent) {
+        child.prototype = Object.create(parent.prototype);
+        child.prototype.constructor = child;
+    }
+}
+
```

Comment:
Can you elaborate on why this is needed? Does something break in the Tethys MapView in newer versions?

Reply:
The "inherits" method had been deprecated and was removed in version OpenLayers version 6, but it is used in the Tethys MapView. See notes [here](https://github.com/openlayers/openlayers/blob/main/changelog/upgrade-notes.md#removal-of-deprecated-methods). I know that allowing users to input their own version will surely one day result in more and more issues with the Tethys MapView, but I figured we could try and cover the ones that are blatantly obvious at this point - which is only this one as far as I found.

Reply:
Makes sense. Sounds like maybe at the ECMA/JavaScript level? As far as supporting future versions, I guess that's the risk the user takes using a version that is not the version we default to, provided we are doing a reasonable job at keeping up to date. Could you add a note to the documentation on this feature that indicates this? Sort of a "use at your own risk" or advise not to update more than minor versions type of thing?